src/Reprocessing.rei

```diff
@@ -21,7 +50,12 @@ module Constants = Reprocessing_Constants;
 
 include Reprocessing_Types.TypesT;
 
-let hotreload: string => bool;
+let hotreload: (~screen: string=?, string) => bool;
+
+let setScreenId: string => unit;
+let clearScreenId: unit => unit;
```

Comment:
Can we add a bit of documentation to those, since we're exposing them and they might actually be useful for people who'd want to integrate reprocessing inside a blog post (which would be amazing if it were super easy)

Reply:
yeah, will do!